testing/resources/docker_hooks_repo/hooks.yaml

```diff
@@ -0,0 +1,11 @@
+-   id: docker-hook
+    name: Docker test hook
+    entry: echo
+    language: docker
+    files: \.txt$
+
+-   id: docker-hook-failing
+    name: Docker test hook with nonzero exit code
+    entry: bork
```

Comment:
Well, it certainly seems useful to base multiple hook entrypoints off the same container.

Happy to make `entry` optional, we'll need to document that people have to define container entrypoints though?